### DISCUSSION TITLE
Image to React code

### DISCUSSION THREAD

**Discussion by Teknofilia** [ngobrolin/web#14] [Ideas]
kali aja web ttg image to code blm dibahas

**Comment by Teknofilia** [comment]
ini contoh webnya [](https://screenshottocode.com/)

**Reply by Teknofilia** [reply]
screenshot(dot)com

**Comment by rizafahmi** [comment]
Belum dibahas detail, baru liat demonya doang kayak [make real tldwar](https://makereal.tldraw.com/) dan https://v0.dev/

Terimakasih idenya 🙏
